### ISSUE TITLE
Stopped working on Chrome

### ISSUE THREAD

**Issue by AKNiS** [ejci/Chrome-Audio-EQ#67]
Hello, 

The sliders still work, but they don't do anything. I've tried various sites, restarted, uninstalled, etc, but it's not working. Is anyone else having this issue?

10.9.5 OS x with Chrome.

**Comment by Ambrace** [comment]
Yep. With Chrome. Firefox. IE. Tried them all.  =/ HTML5ify doesn't do anything to fix that either. =/
